web/admin/src/BigScreen.js

```diff
@@ -131,7 +135,7 @@ export default class BigScreen extends PureComponent {
     })
   }
 
-  renderNonexistent = () => <div className="big-screen"><div className="box box-content">This session has not been initialized for viewing</div></div>
+  renderNonexistent = () => <div className="box"><div className="box-content">This session has not been initialized for viewing</div></div>
```

Comment:
What effect does removing the `big-screen` class have?

Reply:
I moved big screen to the functions that call this to provide more flexibility. big-screen was causing the content cell to take up the entire space

Reply:
Ah, I see now 👍